src/box/sql/build.c

```diff
@@ -59,6 +59,75 @@
 #include "box/user.h"
 #include "box/constraint_id.h"
 #include "box/session_settings.h"
+#include "mpstream/mpstream.h"
+
+/**
+ * Encode format as entry to be inserted to _space on @region.
+ *
+ * @param parse Parse context.
+ * @param def Space definition to encode.
+ * @param[out] size Size of result allocation.
+ *
+ * @retval NULL Error.
+ * @retval not NULL Pointer to msgpack on success.
+ */
+static char *
+encode_format(struct region *region, struct space_def *def, uint32_t *size)
```

Comment:
Having names without `sql_` prefix is fine by me. But keep in mind that you will have to rename it again if you will even need to put it into a header and make non-static.